components/collective-page/hero/Hero.js

```diff
@@ -213,7 +213,7 @@ const Hero = ({ collective, host, isAdmin, onPrimaryColorChange, callsToAction,
                     <StyledLink
                       target="_blank"
                       rel="noopener noreferrer"
-                      href={collective.settings.tos}
+                      href={`terms`}
```

Comment:
I think you meant `/hostCollectiveSlug/terms`

Reply:
Thanks, I don't know I missed this.